modules/geoip-lite-2024-07-19/lib/geoip.js

```
@@ -0,0 +1,540 @@
+			}
+		}
+
+		return null;
+	},
+
+	pretty: function(n) {
+		if (typeof n === 'string') {
+			return n;
+		} else if (typeof n === 'number') {
+			return utils.ntoa4(n);
+		} else if (n instanceof Array) {
```

Biome

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)